Makefile

```diff
@@ -75,7 +75,11 @@ docker: certs build-in-docker pause-container-release cni-plugins .out-stamp
 # 'RELEASE' mode
 # TODO: make this idempotent
 docker-release: pause-container-release cni-plugins .out-stamp
+ifeq ($(TARGET_OS),linux)
```

Comment:
would it be better to use `--build-arg`s? I think this is probably a good time to refactor a bit and make all docker versions homogeneous. 

More info about `build-arg`:
https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg
https://stackoverflow.com/questions/19537645/get-environment-variable-value-in-dockerfile

Reply:
I agree, will update in next revision